src/components/Pages/ArticleDetails/index.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import ArticleModel from '../../ArticleDetails';
+import { VolunteeringBanner } from '../../VolunteeringBanner';
+import { Banner } from '../../ArticleBanner';
+import { ArticlesSearch } from '../../ArticlesSearch';
+const bannerData = {
```

Comment:
why?

Reply:
yes you are right i made a mistake and pushed old commits from another branch
i will fix it and make a new pr